prediction_market_agent/tools/anvil/run.py

```diff
@@ -0,0 +1,64 @@
+import json
+import typing as t
+
+import typer
+from prediction_market_agent_tooling.loggers import logger
+from pydantic import BaseModel
+from web3 import Web3
+
+from prediction_market_agent.agents.microchain_agent.nft_treasury_game.constants_nft_treasury_game import (
+    NFT_TOKEN_FACTORY,
+)
+from prediction_market_agent.agents.microchain_agent.nft_treasury_game.deploy_nft_treasury_game import (
+    DEPLOYED_NFT_AGENTS,
+)
+from prediction_market_agent.db.agent_communication import fetch_unseen_transactions
+from prediction_market_agent.tools.anvil.fetch_metrics import (
+    extract_transactions_involving_agents_and_treasuries,
+    fetch_nft_transfers,
+)
+
+
+def main(rpc_url: str) -> None:
```

Comment:
```suggestion
def main(rpc_url: str, write_output: bool = False) -> None:
```

This way, Typer will automatically expose it as command line argument